almalinux-8.pkr.hcl

```diff
@@ -84,6 +110,7 @@ build {
 
   post-processor "vagrant" {
     compression_level = "9"
+    vagrantfile_template = "tpl/vagrantfile.tpl"
```

Comment:
This template will be applied to all boxes including VirtualBox and VMWare but I don't think it's a good idea to switching them to rsync instead of "native" implementation.

Reply:
I vote for the first variant because of its simplicity. In this particular case, we can have a post-processor for VirtualBox/VMWare (the default template seems to be working fine for now) and another one for Libvirt.

You know, all decisions are reversible, and later it can be refactored if necessary.